src/bokeh/models/widgets/tables.py

```diff
@@ -858,6 +858,47 @@ def __init__(self, *args, **kwargs) -> None:
     The height of each row in pixels.
     """)
 
+    @staticmethod
+    def from_data_table(data, columns=None, formatters={}, **kwargs) -> DataTable:
```

Comment:
I think just `.from(...)`:
```suggestion
    def from(data, columns=None, formatters={}, **kwargs) -> DataTable:
```
`from_data_table` does not seem accurate because the data table is being created *from*: a dict, a dataframe, or a CDS (not from a data table)

Reply:
OK. I understand it now...

Reply:
It would be nice to have arguments to this method statically typed, especially given you've already done part of the work in the docstring. This module isn't checked by mypy, so this doesn't have to strictly type check for now.